### ISSUE TITLE
Unexpected token 'export' when using boilerplate setup code for Remix.

### ISSUE THREAD

**Issue by joeelmahallawy** [radix-ui/themes#227]
I have an app in Remix that's very basic. I tried to follow the setup guide but was getting `Unexpected token 'export`. I literally didn't have anything else other than: 
```import { Theme } from '@radix-ui/themes'
import '@radix-ui/themes/styles.css'
import {
  Links,
  LiveReload,
  Meta,
  Outlet,
  Scripts,
  ScrollRestoration,
} from '@remix-run/react'
import '@unocss/reset/tailwind.css'
import 'virtual:uno.css'

export default function App() {
  return (
    <html lang="en">
      <head>
        <meta charSet="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <Meta />
        <Links />
      </head>
      <body className="font-sans antialiased">
        <Theme>
          <Outlet />
          <ScrollRestoration />
          <Scripts />
          <LiveReload />
        </Theme>
      </body>
    </html>
  )
}
``` 

in my `app/root.tsx` and yet I get that problem.

**Comment by vladmoroz** [comment]
Probably an ESM issue, we have just started investigating this and plan to do a fix within a week or so. Hold on for a bit 🙂
For now you might find a workaround in the issues of this repo if you search it.

**Comment by vladmoroz** [comment]
We released `"@radix-ui/themes": "3.0.0-rc.1"` which should work in ESM projects if you need an early version now. Please refer to the changelog for other changes in this release.
